motor/discovery/gitlab/gitlab.go

```diff
@@ -166,9 +169,17 @@ func (r *Resolver) Resolve(ctx context.Context, root *asset.Asset, pCfg *provide
 						terraformFiles, err := discoverTerraformHcl(ctx, p.Client(), grp.Path, project.Path)
 						if err == nil && len(terraformFiles) > 0 {
 							terraformCfg := pCfg.Clone()
-
-							assets, err := (&terraform_resolver.Resolver{}).Resolve(ctx, projectAsset, terraformCfg, credsResolver, sfn, userIdDetectors...)
+							terraformCfg.Backend = providers.ProviderType_TERRAFORM
+							assets, err := (&terraform_resolver.Resolver{}).Resolve(ctx, projectAsset, terraformCfg, credsResolver, sfn)
 							if err == nil && len(assets) > 0 {
+								for i := range assets {
+									if len(assets[i].PlatformIds) > 0 {
+										assets[i].PlatformIds[0] = assets[i].PlatformIds[0] + "/" + project.Name
```

Comment:
the terraform platform ids in the group i was testing were coming back as the same, so i added the project name as a safeguard..